src/tools/compiletest/src/ferrocene_annotations.rs

```diff
@@ -169,7 +169,87 @@ fn sample_config() -> Config {
         mode: env("FERROCENE_MODE"),
         src_test_suite_root: env("FERROCENE_SRC_BASE"),
         suite: env("FERROCENE_SUITE"),
-        ..Config::default()
+
+        // Dummy values
```

Comment:
Oh no. :( This is quite annoying. Did they remove the deafult config?

Reply:
> Oh no. :( This is quite annoying. Did they remove the deafult config?

there was no sensible default really, so Default should not have been there... maybe we can think of a less painful Config type, where there are groupings of the (so many) fields

Reply:
we should consider https://github.com/rust-lang/rust/issues/143827